services/main/main-server.mjs

```diff
@@ -30,13 +30,18 @@ function mainServer() {
   // let rpcDirectory = new Directory();
   // gatewayService = new GatewayService(rpcDirectory);
 
-  /* Start all services */
+  /* Initialize static asset server */
   staticAssetServer.initialize();
+  /* Start static asset server */
+  staticAssetServer.run();
   // TODO: Uncomment the following when RPC Directory is implemented.
   // gatewayService.initialize();
 
   // TODO: Set the port once configuration is done.
   app.listen(config.get('server.port'));
 }
 
-mainServer();
+if (require.main === module) {
```

Comment:
This is an ES Module, `require` is not available. You might have to try using the es-main package:
https://www.npmjs.com/package/es-main
I've tried it but it doesn't seemed to work well with yarn.